server/src/lib/files.ts

```diff
@@ -279,7 +278,7 @@ export default class Files {
   /**
    * Grab a random sentence and associated sound file path.
    */
-  getRandomClip(uid: string): Promise<string[2]> {
```

Comment:
stating the length of arrays is not actually valid typescript AFAIK.

at the very least, it breaks type checking/type inference.

That said, this method would be better served returning an object (and the signature specified with an inline object type).

I didn't change that here because I didn't have time to look around and properly assess the usages of this method and an appropriate structure and naming of the returned information.

Reply:
agreed, we need an interface definition for this, but no need to worry about that for now.